Userland/Libraries/LibJS/Runtime/Temporal/InstantPrototype.cpp

```diff
@@ -243,24 +243,41 @@ JS_DEFINE_NATIVE_FUNCTION(InstantPrototype::since)
     return TRY(create_temporal_duration(global_object, 0, 0, 0, 0, result.hours, result.minutes, result.seconds, result.milliseconds, result.microseconds, result.nanoseconds));
 }
 
-// 8.3.11 Temporal.Instant.prototype.round ( options ), https://tc39.es/proposal-temporal/#sec-temporal.instant.prototype.round
+// 8.3.11 Temporal.Instant.prototype.round ( roundTo ), https://tc39.es/proposal-temporal/#sec-temporal.instant.prototype.round
 JS_DEFINE_NATIVE_FUNCTION(InstantPrototype::round)
 {
     // 1. Let instant be the this value.
     // 2. Perform ? RequireInternalSlot(instant, [[InitializedTemporalInstant]]).
     auto* instant = TRY(typed_this_object(global_object));
 
-    // 3. If options is undefined, then
+    // 3. If roundTo is undefined, then
     if (vm.argument(0).is_undefined()) {
         // a. Throw a TypeError exception.
         return vm.throw_completion<TypeError>(global_object, ErrorType::TemporalMissingOptionsObject);
     }
 
-    // 4. Set options to ? GetOptionsObject(options).
-    auto* options = TRY(get_options_object(global_object, vm.argument(0)));
+    Object* round_to;
+
+    // 4. If Type(roundTo) is String, then
+    if (vm.argument(0).is_string()) {
+        // a. Let paramString be roundTo.
+        auto& param_string = vm.argument(0).as_string();
```

Comment:
:thinking: since this is used for `CreateDataPropertyOrThrow`, maybe we don't need to yoink out the string just to convert it back to a `Value`?
the `&param_string` seemed a bit odd before I thought about it, is all.

Reply:
Since `CreateDataPropertyOrThrow` copies the Value anyways, I don't think it matters, both will just compile down to a constructor call, but up you

Reply:
I'm not really talking about performance, more so about what the spec comment says vs. what we do:
```c++
// c. Perform ! CreateDataPropertyOrThrow(roundTo, "smallestUnit", paramString).
MUST(round_to->create_data_property_or_throw(vm.names.smallestUnit, &param_string));
```
brain goes: "why the `&`?", then looks above and sees:
```c++
// a. Let paramString be roundTo.
auto& param_string = vm.argument(0).as_string();
```

it doesn't really matter since you either don't even notice it, or realise what's happening by looking at it again, but I thought I'd mention it

Reply:
Ohh I see what you mean, yeah it might be worth it to keep it as a value then.

Reply:
Done, thanks both!